core/pages/Category.js

```diff
@@ -138,7 +138,7 @@ export default {
     }
     if (!isServer && this.lazyLoadProductsOnscroll) {
```

Comment:
we could somehow try to extract it to some module helper or mixin to be used on more placew without wondering about listener, or is it on server so wouldn't have window property - devs could just import it and use and react on it then it's changed